spark2-sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1402,14 +1402,6 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
-  val FORCE_SHIMCALLER_CLASSLOADER = conf("spark.rapids.force.caller.classloader")
```

Comment:
Does anything else in spark2 use this?

Reply:
This is merely due to spark2 cloning RapidsConf source. We verified there is no actual use of spark.rapids.force.caller.classloader in the spark 2 explain support.